index.html

```diff
@@ -187,8 +167,7 @@ <h1>Prototype Viewer MediaWiki 3D GLTF Data, STL data is already available </h1>
             document.getElementById('container').appendChild(this.renderer.domElement);
             this.manager = new THREE.LoadingManager();
             // camera
-            this.camera = new THREE.PerspectiveCamera(60, dimensions.ratio, 0.001, 500000);
-            this.camera.up.set(0, 0, 1);
+            this.camera = new THREE.PerspectiveCamera(60, dimensions.ratio);
```

Comment:
The near/far planes are assigned in `stageContent()` now, so I've just left the defaults here.

We probably want to avoid changing the camera's default 'up' vector, I think it's better to rotate the content to match a Y-Up convention (if a conversion is needed at all). See later section on this.